### ISSUE TITLE
How does the labels of DBLP come from?

### ISSUE THREAD

**Issue by hcmdgh** [shenxiaocam/ACDNE#1]
Hello! In your work you use Citationv1, DBLPv7 and ACMv9 datasets, but i havent found labels in them. So how do you get labels?

**Comment by shenxiaocam** [comment]
Hi, in the ".mat" file, the variable "group" represents a node label matrix. 
"group" is a n*c matrix, where n is the number of nodes, and c is the number of node label categories.

**Comment by hcmdgh** [comment]
> Hi, in the ".mat" file, the variable "group" represents a node label matrix. "group" is a n*c matrix, where n is the number of nodes, and c is the number of node label categories.

Yes, I have noticed the file. But how do you calculate the labels? In such raw datasets I dont find any labels.
What's more, can you tell me the papers' years you use in dataset?

**Comment by shenxiaocam** [comment]
We labeled each paper according to its research topics (i.e. the conference or journal it is published). The five labels include “Databases”, “Artificial Intelligence”, “Computer Vision”, “Information Security”, and “Networking”. 
The DBLPv7, Citationv1 and ACMv9 we extracted were between years 2004 and 2008, before year 2008, and after year 2010.
